src/drivers/frsky_telemetry/sPort_data.c

```diff
@@ -302,13 +317,31 @@ void sPort_send_GPS_CRS(int uart)
 	/* send course */
 
 	/* convert to 30 bit signed magnitude degrees*6E5 with MSb = 1 and bit 30=sign */
-	uint32_t iYaw = 100 * global_pos->yaw;
+	int32_t iYaw = local_pos->yaw * 18000.0f / M_PI_F;
```

Comment:
It's a good question that maybe someone can help me with. For telemetry we are really looking for instantaneous data like in the VFR HUD mavlink message. I found during testing that global position data seems to not be provided for some time after start up and therefore gives no indications, even though the mavlink messages do.  local position seems to provide the data more quickly for heading. I haven't been able to really work out the difference between the different sources. If someone can point me in the right direction to find the appropriate message source I would appreciate it.